cli/script/command-executor.ts

```diff
@@ -1289,5 +1296,16 @@ function whoami(command: cli.ICommand): Promise<void> {
     return sdk.getAccountInfo()
         .then((account): void => {
             log(`${account.email} (${account.linkedProviders.join(", ")})`);
+
+            var connectionInfo = deserializeConnectionInfo();
+            if (!connectionInfo.noProxy && connectionInfo.proxy) {
+                log(`Proxy: ${connectionInfo.proxy}`);
```

Comment:
Should we display something if the `noProxy` flag was set? Otherwise, the user might get confused why CodePush isn't using a system-wide setting. Maybe we could display something like "Proxy: Ignored" to indicate this?